src/modules/commentModal/getComments.js

```diff
@@ -0,0 +1,16 @@
+'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/yOdv0JXV8ssnN9f9thVW/comments';
+
+// const getCount = (id) => comments.length;
+
+const getComments = async (id) => {
+  const response = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/yOdv0JXV8ssnN9f9thVW/comments?item_id=${id}`, {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+    },
+  });
+  const jsonResponse = await response.json();
+  return jsonResponse;
+};
+
+export default getComments;
```

Comment:
Please remember to consume data from the API for this function :smile:

Reply:
Hi Romina. 
Thanks for the feedback.
 Refactored it. It now works.